### DISCUSSION TITLE
Welcome to ZEN-garden Discussions!

### DISCUSSION THREAD

**Discussion by johburger** [ZEN-universe/ZEN-garden#85] [ZEN-garden announcements]
## 👋 Welcome to the ZEN-garden discussions!
  We’re using Discussions as a place to host FAQs, discuss new ideas and document How-Tos. We hope that you:
  * Ask questions you’re wondering about.
  * Share issues you're having while setting up the model.
  * Contribute to the How-To documents so that new members have a smoother start.
  * Help with shaping this discussions forum and give feedback for improvements. 

Have fun and happy coding 👩‍💻!
Cheers, 
🦖

**Comment by aganter** [comment]
For a quick overview please take a look at the presentation: [Presentation ZEN garden.pptx](https://github.com/RRE-ETH/ZEN-garden/files/8334880/Presentation.ZEN.garden.pptx)
